readme.md

```diff
@@ -260,6 +260,16 @@
 | ------: | -------: | -----: |
 |    2    |     5    | 5-120m |
 
+### [Commonspoly](https://commonspoly.cc/)
+
+> Commonspoly is a fun strategy and turn based RPG game where players of the game work together to defend their world from being taken over by speculators. The game is like the direct opposite of what one does in the game of Monopoly. For in Commonspoly owning private goods is frowned upon and converting the privately owned goods to either public or community owned ones will be the task of the players. The game is really fun as its a pit against good and evil, where evil is/are depicted by the speculator characters in the game. Speculators go around the game and try to undo the efforts of the defendors. Also the game is released under Peer Production License and can be downloaded for free and printed at home to play.
+
+<img src="https://upload.wikimedia.org/wikipedia/commons/b/bc/Commonspoly.JPG" alt="Commonspoly game image" width="250"/>
```

Comment:
Per [the formatting guide](https://github.com/edm00se/awesome-board-games/blob/master/formatting.md), the image should be ~250px wide. While your html image tag achieves this with setting the width attribute, it still loads an overly large image.

Since the file is hosted by wikimedia, finding the image resource page means we can view other prepared sizes of the image. There's one of 320px ready to go, which is far more preferable.
https://commons.wikimedia.org/wiki/File:Commonspoly.JPG

Additionally, the formatting guide requests the image to be done in the markdown format. All in all, the above should look like:

```suggestion
![Commonspoly game image](https://upload.wikimedia.org/wikipedia/commons/thumb/b/bc/Commonspoly.JPG/320px-Commonspoly.JPG)
```

Reply:
Thanks for suggestions @edm00se , I will make the changes and update the pull request.